docs/05_Conversation_History.ipynb

```diff
@@ -0,0 +1,138 @@
+{
```

Comment:
<p>Please avoid leaving the <code>sys.path</code> hacks in the notebook when you raise the PR. Remove them manually, or update your environment so that you don't need them anymore. </p><p> </p><p>Repeated imports of <code>os</code> and <code>sys</code> as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/agentai/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Okay.